app/views/bus_stops/edit.html.haml

```diff
@@ -1,3 +1,3 @@
-%h1.page-title Editing #{ display_name @stop }
+%h1.page-title Editing #{ @stop.name }
```

Comment:
We're here in the edit view, so it's definitely a persisted stop, and `name` is validated to be present (and it's not actually user-editable).